LICENSE.md

```diff
@@ -0,0 +1,45 @@
+# Royalty-free Software provided by Siemens on sharing platforms for developers/users of Siemens products 
+
+# 1. General information: Software in source code and object code  
```

Comment:
In general all markdown files should have only one H1 headline to follow the Markdown best practices. https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md

Reply:
See rule 025. https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md025---multiple-top-level-headings-in-the-same-document 

Recomend to use https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint to verify the License text for erros.

Reply:
thanks for the hint, I just copied the file I received from legal. I adapted the license file regarding the rules